appinventor/codelabs/talk-to-me/index.html

```diff
@@ -0,0 +1,163 @@
+
+<!doctype html>
+
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <meta name="theme-color" content="#4F7DC9">
+  <meta charset="UTF-8">
+  <title>Talk To Me Codelab</title>
+  <script src="../../bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
+  <link rel="import" href="../../elements/codelab.html">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Code+Pro:400|Roboto:400,300,400italic,500,700|Roboto+Mono">
+  <link rel="icon" href="../../elements/favicon.png">
+  <style>
+    body {
+      font-family: "Roboto",sans-serif;
+    }
+  </style>
+
+</head>
+<body unresolved class="fullbleed">
+
+  <google-codelab title="Talk To Me Codelab" feedback-link="https://goo.gl/forms/9zDi1V3arg167lIY2">
+
+      <google-codelab-step label="Overview">
+        <p>In this codelab, you&#39;ll learn how to make your first app!
+        <h3 class="checklist">What you&#39;ll learn</h3>
+        <ul class="checklist">
+          <li>Use the <b>Designer View</b> to create the user interface.</li>
+          <li>Use the <b>Blocks View</b> to add functionality to the app.</li>
+          <li>Use the <b>Button</b> and <b>TextToSpeech</b> components.</li>
+          <li>Use <b>Event Handlers</b>.</li>
+        </ul>
+
+
+      </google-codelab-step>
+
```

Comment:
@amyzhaang Since this is the first codelab, can you add a step that walks through how to set up an App Inventor account?